api-docs/list-clients.md

```diff
@@ -0,0 +1,17 @@
+#list-clients API
+#POST Request
+#URL /api/list-clients/
+#Parameters: {firstName, LastName}
```

Comment:
This documentation doesn't help me understand what I actually need to send in the request. Parameters probably refers to query parameters, but I'm not sure whether to send the parameters with `&` in between them or `,` in between them. I find that the best documentation has code examples. Here's an example of what I mean.

```markdown
## Request
GET /clients?firstName=Mario&lastName=Luigi&zip=84109&page=1

Notes
- The values for the query parameters should be [URL encoded](https://www.w3schools.com/tags/ref_urlencode.asp)
- The `page` query parameter defaults to 1.
- The `zip` query parameter can be a partial zip -- the backend algorithm does a "startsWith" check

## Response
```
```json
{
  "clients": [
    {
      "firstName": "Leonel",
      "lastName": "Nieto",
      "dob": "2019/05/30",
      "zip": "84109-0012",
      "primaryPhone": "8015550888",
      "addedBy": {
        "id": 5,
        "name": "Joel Denning"
      },
      "dateAdded": 123123323222,
    }
  ],
  "pagination": {
    "currentPage": 1,
    "totalPages": 10,
    "pageSize": 100,
    "totalNumClients": 954
  }
}
```
```markdown
Notes
- `dateAdded` is a unix timestamp.
```